ShareX.UploadersLib/Forms/UploadersConfigForm.cs

```diff
@@ -2044,6 +2060,252 @@ private void txtPomfResultURL_TextChanged(object sender, EventArgs e)
+
+            if (!string.IsNullOrEmpty(username) && !string.IsNullOrEmpty(password))
+            {
+                try
+                {
+                    Seafile sf = new Seafile(txtSeafileAPIURL.Text, null, null);
+                    string authToken = sf.GetAuthToken(username, password);
+
+                    if (!string.IsNullOrEmpty(authToken))
+                    {
+                        txtSeafileUsername.Text = "";
+                        txtSeafilePassword.Text = "";
+                        Config.SeafileAuthToken = authToken;
+                        txtSeafileAuthToken.Text = authToken;
+                        btnRefreshSeafileAccInfo.PerformClick();
+                        Config.SeafileRepoID = sf.GetOrMakeDefaultLibrary(authToken);
+                        txtSeafileUploadLocationRefresh.PerformClick();
+                        MessageBox.Show(Resources.UploadersConfigForm_Login_successful, "ShareX", MessageBoxButtons.OK, MessageBoxIcon.Information);
+                    }
+                    else
+                    {
+                        MessageBox.Show(Resources.UploadersConfigForm_Login_failed, "ShareX", MessageBoxButtons.OK, MessageBoxIcon.Error);
+                    }
+                }
+                catch (Exception ex)
+                {
+                    DebugHelper.WriteException(ex);
+                    MessageBox.Show(ex.ToString(), Resources.UploadersConfigForm_Error, MessageBoxButtons.OK, MessageBoxIcon.Error);
+                }
+            }
+        }
+
+        private void cbSeafileCreateShareableURL_CheckedChanged(object sender, EventArgs e)
+        {
+            Config.SeafileCreateShareableURL = cbSeafileCreateShareableURL.Checked;
+        }
+
+        private void cbSeafileIgnoreInvalidCert_CheckedChanged(object sender, EventArgs e)
+        {
+            Config.SeafileIgnoreInvalidCert = cbSeafileIgnoreInvalidCert.Checked;
+        }
+
+        private void btnRefreshSeafileAccInfo_Click(object sender, EventArgs e)
+        {
+            if (string.IsNullOrEmpty(txtSeafileAuthToken.Text) || string.IsNullOrEmpty(txtSeafileAPIURL.Text))
+            {
+                return;
+            }
+
+            Seafile sf = new Seafile(txtSeafileAPIURL.Text, txtSeafileAuthToken.Text, null);
+            Seafile.SeafileCheckAccInfoResponse SeafileCheckAccInfoResponse = sf.GetAccountInfo();
+
+            if (SeafileCheckAccInfoResponse == null)
+            {
+                MessageBox.Show(Resources.UploadersConfigForm_Login_failed, "ShareX", MessageBoxButtons.OK, MessageBoxIcon.Error);
+                return;
+            }
+            txtSeafileAccInfoEmail.Text = SeafileCheckAccInfoResponse.email;
+            txtSeafileAccInfoUsage.Text = HelpersLib.NumberExtensions.ToSizeString(SeafileCheckAccInfoResponse.usage).ToString() + " / " + HelpersLib.NumberExtensions.ToSizeString(SeafileCheckAccInfoResponse.total).ToString();
```

Comment:
I don't understand why you used extensions like that in here. Not to mention you converted string to string there.
Extensions used like this: ```SeafileCheckAccInfoResponse.usage.ToSizeString()```